app.js

```diff
@@ -52,115 +31,38 @@ app.get('/sensorData/:days', async (req, res) => {
   const dateNow = new Date(myDate).toISOString();
```

Comment:
om du har tid, hadde det vært kjekt om du kunne ha lagt datoformatteringsfunksjonen i en egen hjelper. Vi mangler tidssone-sted i funksjonen jeg påbegynte men ikke ferdigstilte.

Reply:
Har de variablene noe å si? Ignorerte de i siste commit..

Reply:
Tenkte du at vi skal bruke de i responsen?

Reply:
Det var mer ifm med å lagre datoene i samme format som tidligere når vi POST-er, for å ikke brekke Data Studio